utils/python/CIME/XML/GenericXML.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, infile=None):
             self.filename = infile
             self.root = ET.Element(infile)
             self.root.set('version','1.0')
-            
```

Comment:
I just added this option to my emacs:
(add-hook 'before-save-hook 'delete-trailing-whitespace)